glean-core/csharp/Glean/Glean.cs

```diff
@@ -76,16 +90,16 @@ string dataDir
                 return
             }
 
-            this.applicationContext = applicationContext
+            this.applicationContext = applicationContext*/
+
+            this.configuration = configuration;
+            httpClient = new BaseUploader(configuration.httpClient);
```

Comment:
IIRC, `configuration.httpClient` here is null?

Reply:
Yes, that's fine. This PR will not provide any HTTP uploader, it will only create the upload manager.